### PR TITLE
Tests deployment

### PR DESCRIPTION
**Pull request by TRBaldim**
I'm Adding few more details for the CI perspective. I had much work related to Implement some tests and think about what can we do related to the Hero and the other instances. I think that can help a lot with that.

Please take a look 🚀

### CODE REVIEW

**Comment by TRBaldim** [comment]
[Build the Integration with Travis](https://trello.com/c/CbE5fe5J/14-build-the-integration-with-travis)

**Comment by TRBaldim** [comment]
[Build tests for the Prototype](https://trello.com/c/cK01QqNj/15-build-tests-for-the-prototype)

File: src/main/scala/com/thebrauproject/elements/package.scala [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 
   case class Hero(id: String,
                   name: String,
+                  createAt: Timestamp,
+                  modifiedAt: Timestamp,
                   skills: Seq[Skill],
-                  role: Role) extends Creature
+                  roleId: String,
+                  deleted: Boolean
+                 ) extends Creature
```
Comment (andre250) [inline src/main/scala/com/thebrauproject/elements/package.scala]: In this case you are declaring all the hero fields ?

File: src/main/scala/com/thebrauproject/elements/package.scala [hunk 2]
```diff
@@ -23,7 +29,7 @@ package object elements {
 
   case class Task(title: String,
                   text: String,
-                  skill: Seq[Skill]) extends Objective
+                  skill: Seq[Skill]) extends Action
```
Comment (andre250) [inline src/main/scala/com/thebrauproject/elements/package.scala]: Didnt get why a array of skills will extend action

File: src/main/scala/com/thebrauproject/util/package.scala [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import java.time.Instant
+
+import com.google.common.io.BaseEncoding
+
+package object util {
+
+  def utc: Timestamp = Timestamp.from(Instant.now)
+
+  def Base64SHA256(string: String): String =
+    BaseEncoding
```
Comment (andre250) [inline src/main/scala/com/thebrauproject/util/package.scala]: nice call!

**Pull request merged**